app/controllers/bookings_controller.rb

```diff
@@ -0,0 +1,22 @@
+class BookingsController < ApplicationController
+  def create
+    @booking = Booking.new
+    @user = current_user
+    @lesson = Lesson.find(params[:lesson_id])
+    @booking.lesson = @lesson
+    @booking.user = @user
+    @booking.save!
+    if current_user
+      @booking.save!
```

Comment:
Should replace with test on successfully saving ; if not, re-render page.
Possible cases = already booked, conflict with other booked lesson (--> need extra validations)